build.sbt

```diff
@@ -701,16 +751,18 @@ lazy val scalaDist = Project("scala-dist", file(".") / "target" / "scala-dist-di
   )
   .dependsOn(libraryAll, compiler, scalap)
 
-lazy val root = (project in file("."))
+lazy val root: Project = (project in file("."))
   .settings(disableDocs: _*)
   .settings(disablePublishing: _*)
   .settings(generateBuildCharacterFileSettings: _*)
   .settings(
     publish := {},
     publishLocal := {},
     commands ++= ScriptCommands.all,
-    Quiet.silenceIvyUpdateInfoLogging
-)
+    Quiet.silenceIvyUpdateInfoLogging,
+    antStyle := false,
+    incOptions := incOptions.value.withNameHashing(!antStyle.value).withAntStyle(antStyle.value)
```

Comment:
I used to have a separate `local.sbt` (which I keep in my global `~/.gitignore`) with the following line:
```
(incOptions in ThisBuild) := (incOptions in ThisBuild).value.withNameHashing(false).withAntStyle(true)
```

This gives an error now:
```
> compile
...
[error] (library/*:incOptions) java.lang.AssertionError: assertion failed: Name hashing and Ant-style cannot be enabled at the same time.
...
```

I tried putting `antStyle := true` in my `local.sbt`, but that doesn't work (`local.sbt:1: error: not found: value antStyle`). What is the correct way to enable it? I wouldn't like to type `set antStyle := true` each time I start sbt.